lib/ammeter/rspec/generator/example.rb

```diff
@@ -8,14 +9,15 @@ def c.escaped_path(*parts)
 
   generator_path_regex = c.escaped_path(%w[spec generators])
   if RSpec::Core::Version::STRING >= '3'
+    c.include Ammeter::RSpec::Rails::GeneratorExampleHelpers
```

Comment:
I think you want to add `:type => :generator` here so these helpers are only added to generator specs